cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java

```diff
@@ -174,14 +184,14 @@ public void setVoltageAngleNodeBreaker() {
         }
         Bus bus = t.getBusView().getBus();
         if (bus == null) {
-            bus = t.getBusBreakerView().getBus();
+            bus = t.getBusBreakerView().getConnectableBus();
```

Comment:
We are looking for a bus to set a voltage associated to this terminal. We must look for a connected bus in the IIDM network. If we obtain the connectable bus we are assigning the voltage to a different electric bus.

Reply:
Okay, I reverted the change. I did not pay attention to the fact that if the bus was disconnected, the connectable bus was calculated to get a different one.